manifests/b/Bisq/Bisq/1.7.3/Bisq.Bisq.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Bisq.Bisq
+PackageVersion: 1.7.3
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://github.com/qishibo/AnotherRedisDesktopManager/releases/download/v1.4.9/Another-Redis-Desktop-Manager.1.4.9.exe
```

Comment:
I believe something went wrong when creating the manifest. This seems to be half of `Bisq` and half of `Another Redis Desktop Manager`.

Reply:
Bad day to submit PRs i guess 😵